CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - Available for `UnusedConstant`, `UnusedField`, `UnusedGlobalVariable`, `UnusedProperty`,
     `UnusedRoutine`, and `UnusedType`.
   - Excludes public API (declared with public visibility in the interface section).
+- `ShortString` and `AnsiString` overloads for `Length` intrinsic.
```

Comment:
I think this should be in the "Fixed" section. Very technically they're additions, but in practice it's a fix for something we were modelling incorrectly (just like the other things in this PR that have been put under "Fixed").

Reply:
Didn't mean to complete the review so fast 🤦 I'm still reviewing.

Reply:
Makes sense to me. 👍